packages/preview/modernpro-cv/1.0.0/template/cv-single.typ

```diff
@@ -0,0 +1,87 @@
+#import "@preview/modernpro-cv:1.0.0": *
+
+#show: cv-single.with(
+  font-type: "PT Serif",
+  continue_header: "false",
+  name: "John Doe",
+  address: "123 Street, City, Country",
+  lastupdated: "true",
+  pagecount: "true",
+  date: "2024-07-03",
+  contacts: (
+    (text: "08856", link: ""),
+    (text: "example.com", link: "https://www.example.com"),
+    (text: "github.com", link: "https://www.github.com"),
+    (text: "123@example.com", link: "mailto:123@example.com"),
+  ),
+)
+
+// about
+#section[about]
+#descript[#lorem(50)]
+#sectionsep
+#section("Education")
+#education(institution: [#lorem(4)], major: [#lorem(2)], period: "xxxx-xxxx", location: "UK", description: [
```

Comment:
I think the `period` argument was renamed to `date`.

Reply:
Hi,

Many thanks for checking!

I have corrected the arguments in the template and updated it. 

Thanks!